redskyctl/internal/commands/grant_permissions/generator.go

```diff
@@ -67,41 +75,76 @@ func NewGeneratorCommand(o *GeneratorOptions) *cobra.Command {
 func (o *GeneratorOptions) addFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&o.SkipDefault, "skip-default", o.SkipDefault, "Skip default permissions.")
 	cmd.Flags().BoolVar(&o.CreateTrialNamespaces, "create-trial-namespace", o.CreateTrialNamespaces, "Include trial namespace creation permissions.")
+	cmd.Flags().StringVar(&o.NamespaceSelector, "ns-selector", o.NamespaceSelector, "Bind to matching namespaces.")
```

Comment:
I think we'd also want to support selecting namespaces by name.

Reply:
I was thinking about it, but I wasn't sure what it would look like.